testsuite/MDAnalysisTests/lib/test_nsgrid.py

```diff
@@ -371,3 +376,35 @@ def test_issue_2670():
 
     # should return the one atom overlap
     assert len(ag2.select_atoms('around 0.0 resid 3')) == 1
+
+
+def high_mem_tests_enabled():
```

Comment:
I think you can do something like this here: https://stackoverflow.com/a/66973011
But honestly whatever works - I wouldn't bother changing this unless you feel like playing around with pytest.